docs/tutorials/smart-rollup/run.md

```diff
@@ -102,14 +102,14 @@ The workflow for deploying to a testnet is similar to the workflow that you used
 1. Check the log file
 -->
 
-To continue your work with smart rollups, you can you can explore examples from the [kernel gallery](https://gitlab.com/tezos/kernel-gallery/-/tree/main/) or create your own.
+To continue your work with Smart Rollups, you can you can explore examples from the [kernel gallery](https://gitlab.com/tezos/kernel-gallery/-/tree/main/) or create your own.
 
 ## References
 
```

Comment:
@timothymcmackin Should we add the tezos.com/smart-rollups site here first or second ?

Reply:
When I get into smart rollups I'll look at the linking and maybe link from here to the [smart rollups doc page](https://docs.tezos.com/docs/architecture/smart-rollups).